### PR TITLE
added news for 2024-09-08

### PR DESCRIPTION
**Pull request by CaqKa**
Manualy added the news for 2024-09-08, checked with zola serve before hand.

### CODE REVIEW

File: content/news/2024-09-08/405/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+2024.09.08 release has been sent to review.
+- **Track Recorder:** Record and save your routes while hiking, biking, running, or navigating.
+- **OpenStreetMap:** Swithched to browser-based authentication to comply with google play on Android.
+- **New Map Data:** Updated with data as of September 7, including isolines for Egypt and Algeria.
+- **Translations:** Enhanced Chinese, Hebrew, and Italiand, and added translations for trail difficulty.
+
+Get the new release first before it gets approved Apple/Google:
+- iOS:  [TestFlight](https://testflight.apple.com/join/lrKCl08I)
+- Android: see APKs attached in Telegram channel
```
Comment (vng) [inline content/news/2024-09-08/405/index.md:17]: Put the link on GitHub release here?

File: content/news/2024-09-08/405/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Detailed changes can be seen on [GitHub](https://github.com/organicmaps/organicmaps/releases/tag/2024.09.08-7-android).
```
Comment (vng) [inline content/news/2024-09-08/405/index.md:19]: Github release page is not detailed, but it refers to the site for the detailed changelog itself :)

Actually, we are supposed to have the best-detailed changelog on-site.
Maybe I can take a look later and will it with more info based on the commits range ..

**Review by vng — COMMENTED** [review]
It works!

File: content/news/2024-09-08/405/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  news: ["Releases"]
+---
+
+2024.09.08 release has been sent to review.
+- **Track Recorder:** Record and save your routes while hiking, biking, running, or navigating.
+- **OpenStreetMap:** Swithched to browser-based authentication to comply with google play on Android.
+- **New Map Data:** Updated with data as of September 7, including isolines for Egypt and Algeria.
+- **Translations:** Enhanced Chinese, Hebrew, and Italiand, and added translations for trail difficulty.
+
+Get the new release first before it gets approved Apple/Google:
```
Comment (pastk) [inline content/news/2024-09-08/405/index.md:15]: 2 months later it doesn't make sense any more, right? :)

So its better to reduce this news item to the bare changelog.

**Pull request closed without merging**